modules/is_analytics_dashboard/manifests/init.pp

```diff
@@ -46,53 +33,108 @@
     home   => "/home/${user}",
     system => true,
   }
-  # Ensure the installation directory is available
-  file { "/opt/${product}":
-    ensure => 'directory',
-    owner  => $user,
-    group  => $user_group,
+
+  /*
+  * Java Distribution
+  */
+
+  # Copy JDK distrubution path
+  file { "${$lib_dir}":
+    ensure => directory
   }
 
-  # Copy the installer to the directory
-  file { "/opt/${product}/${product_package}":
-    owner  => $user,
-    group  => $user_group,
-    mode   => '0644',
-    source => "puppet:///modules/${module_name}/${product_package}",
+  # Copy JDK to Java distribution path
+  file { "jdk-distribution":
+    path  =>  "${java_home}.tar.gz",
+    source => "puppet:///modules/${module_name}/${jdk_name}.tar.gz",
```

Comment:
Shall we format the code?

Reply:
Resolved https://github.com/wso2/puppet-is/pull/60/commits/4db1eb3bd9b5f960a543b0b32012f23fbd0e2659